.spelling

```diff
@@ -6,6 +6,8 @@
 1KiB
 200-tab
 28-day
+2FA
+5a
```

Comment:
Where is this used?

Reply:
2FA is used in some places in `src/datasets/fxa_metrics/funnels.md`, 5a was a numbering convention, switching it to "5-a" seems to have made the tests pass again so ill remove it from the exception list.